keda/values.yaml

```diff
@@ -24,6 +24,26 @@ image:
 # -- Kubernetes cluster domain
 clusterDomain: cluster.local
 
+verticalPodAutoscaler:
```

Comment:
Let's use `autoscaling.verticalPodAutoscaler` as a prefix in case we add other options

Reply:
Done